knowledge-base/report-server-credentials-provided-to-report-viewer-are-displayed-as-plain-text.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Report server credentials provided to report viewer are displayed as plain text
+description: Securing report server domain, username and password used in report viewer's initialization
+type: troubleshooting
+page_title: Report server credentials provided to report viewer are visible on examining the web page source
+slug: report-server-credentials-provided-to-report-viewer-are-displayed-as-plain-text
+position: 
+tags: HTML5Viewer, ReportServer
+ticketid: 1389398
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+How to prevent users from accessing the server with the credentials available while examining the Report Viewer's web page source?
+
+## Solution
+When using HTML5 Report Viewer to display reports stored on Telerik Report Server the report server credentials can be seen as plain text if you examine the web page source code. 
```

Comment:
Credentials should not be provided as plain text. Either use the Guest user or ask the user of the web page to fill in his credentials before they can access the report.

Reply:
The changes were applied